.github/workflows/buildAndPublish.yml

```diff
@@ -0,0 +1,57 @@
+name: Build and Publish @extrahorizon/exh-cli
```

Comment:
Proof that it works : https://github.com/ExtraHorizon/exh-cli/actions/runs/7209757817/job/19641390550